src/render/decorations/CHyprDropShadowDecoration.cpp

```diff
@@ -69,6 +69,8 @@ void CHyprDropShadowDecoration::draw(CMonitor* pMonitor, float a) {
         return; // disabled
 
     // get the real offset
+    // TODO: please tell me you didn't meant to do this every frame (in a try catch block???)
```

Comment:
classic example of me saying "I'll fix this later".

Reply:
It's easy to fix!
I'll leave that to you.

Reply:
ik sure sec

Reply:
done